my-dev-portal/src/components/stripe/Auth0StripeProducts.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
simliarly here. maybe AuthoStripeProducts and OktaStripeProducts can be combined? and just pass down prop like. userAuthinfo ? user Object?